code/modules/mob/living/simple_animal/friendly/mouse.dm

```diff
@@ -34,13 +34,28 @@
 	holder_type = /obj/item/holder/mouse
 	can_collar = 1
 	gold_core_spawnable = CHEM_MOB_SPAWN_FRIENDLY
+	var/chew_probability = 1
 
 /mob/living/simple_animal/mouse/Initialize(mapload)
 	. = ..()
 	AddComponent(/datum/component/squeak, list('sound/creatures/mousesqueak.ogg' = 1), 100)
 
 /mob/living/simple_animal/mouse/handle_automated_speech()
 	..()
+	if(prob(chew_probability))

```

Comment:
This isn't speech! Use the right one, please. `handle_automated_action`

Reply:
👄